test/tools/ossfuzz/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_custom_target(ossfuzz)
+add_dependencies(ossfuzz solc_opt_target solc_noopt_target const_opt_target)
+
+add_executable(solc_opt_target solc_opt_target.cpp)
```

Comment:
I think the name of the executable should contain `ossfuzz` somehow.

Reply:
How about the following convention: `<binary_name>_ossfuzz` e.g., `solc_opt_ossfuzz`?

Reply:
Sounds good!